client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -116,18 +116,32 @@
           </div>
           <dp-editor
             id="addRecommendationTipTap"
-            editor-id="recommendationText"
             v-model="actions.addRecommendations.text"
+            editor-id="recommendationText"
+            editorInsertAtCursorPos
```

Comment:
Since the prop was removed, I removed it also here. Done in[3c4181b](https://github.com/demos-europe/demosplan-core/pull/875/commits/3c4181bcd4ca37b23fed033bc60cc770e760a973)